README.md

```diff
@@ -241,7 +241,7 @@ The `--pattern` flag accepts a Ruby glob argument, which the script will interna
 
 ### Allow Telegraf to access PE-PostgreSQL
 
-The following class is required to be applied to the Master (or the PE Database node if using external PostgreSQL) for collection of PostgreSQL metrics via Telegraf.
+The following class is required to be applied to the Primary Server (or the PE Database node if using external PostgreSQL) for collection of PostgreSQL metrics via Telegraf.
 
 ```puppet
 node 'master.example.com' {
```

Comment:
There are several instances of master being used as a sample node name, i would find and change all of these as well